exchanges/gateio/gateio_websocket.go

```diff
@@ -58,15 +60,15 @@ const (
 var defaultSubscriptions = subscription.List{
 	{Enabled: true, Channel: subscription.TickerChannel, Asset: asset.Spot},
 	{Enabled: true, Channel: subscription.CandlesChannel, Asset: asset.Spot, Interval: kline.FiveMin},
-	{Enabled: true, Channel: subscription.OrderbookChannel, Asset: asset.Spot, Interval: kline.HundredMilliseconds},
+	{Enabled: true, Channel: subscription.OrderbookChannel, Asset: asset.Spot, Interval: kline.TwentyMilliseconds},
```

Comment:
🚧 Sorry buddy. This will require an config upgrade.
I have 2 branches in flight that improve the ease of doing this, so I can provide a cherry-pick to make this easier.
You can also solo it however you want.

Reply:
Reverted change: [c338bc6](https://github.com/thrasher-corp/gocryptotrader/pull/1863/commits/c338bc6a41fe33636b24fa3f3b9e588e2d36be54)